<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomestayUpdate</class>
 <widget class="QWidget" name="HomestayUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Homestay Management - Update Homestay</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLoc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxType">
        <item>
         <property name="text">
          <string>Apartment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Condominium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Villa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bungalow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Terrace</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number of Bedrooms</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditBed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Number of Bathrooms</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditBath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Carpark(s)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditCar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditPrice"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>220</height>
       </size>
      </property>
      <property name="title">
       <string>AMENITIES</string>
      </property>
      <widget class="QCheckBox" name="checkBoxSmoke">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Non-smoking</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxWifi">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Wifi</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonHome">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConfirm">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>lineEditBed</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>lineEditBath</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>lineEditCar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>clicked()</signal>
   <receiver>lineEditPrice</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxSmoke</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReset</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxWifi</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
